3dhubs_amsterdam_python_flask.html

```diff
@@ -0,0 +1,64 @@
+---
+title:  Python Developer
+company: 3D Hubs
+url: https://www.3dhubs.com/
+location: Amsterdam, the Netherlands
+contract: temporary but can be converted to permanent 
+contact:
+    name: Mrs A. Cojanu
+    email: alina@3dhubs.com
+    phone: 0031 6 81 11 41 17
+created: !!timestamp '2017-01-27' 
+tags:
+  - the Netherlands
```

Comment:
Tags should be single words. You can have "Netherlands", but you really don't need it because you should provide this information as a location.